apps/api-gateway/src/connection/connection.controller.ts

```diff
@@ -184,36 +184,39 @@ export class ConnectionController {
    * @param orgId
    * @returns Callback URL for connection and created connections details
    */
-    @Post('wh/:orgId/connections/')
-    @ApiExcludeEndpoint()
-    @ApiOperation({
-        summary: 'Catch connection webhook responses',
-        description: 'Callback URL for connection'
-    })
-    @ApiResponse({ status: HttpStatus.CREATED, description: 'Created', type: ApiResponseDto })
-    async getConnectionWebhook(
-        @Body() connectionDto: ConnectionDto,
-        @Param('orgId') orgId: string,
-        @Res() res: Response
-    ): Promise<Response> {
-        connectionDto.type = 'Connection';
-        this.logger.debug(`connectionDto ::: ${JSON.stringify(connectionDto)} ${orgId}`);
-
-        // const webhookUrl = await this.connectionService._getWebhookUrl(connectionDto.contextCorrelationId);
-
-        // if (webhookUrl) {
-        //   try {
-        //     await this.connectionService._postWebhookResponse(webhookUrl, { data: connectionDto });
-        // } catch (error) {
-        //     throw new RpcException(error.response ? error.response : error);
-        // }
-        const connectionData = await this.connectionService.getConnectionWebhook(connectionDto, orgId);
-        const finalResponse: IResponse = {
-            statusCode: HttpStatus.CREATED,
-            message: ResponseMessages.connection.success.create,
-            data: connectionData
-        };
-
-        return res.status(HttpStatus.CREATED).json(finalResponse);
-    }
-}     
+  @Post('wh/:orgId/connections/')
+  @ApiExcludeEndpoint()
+  @ApiOperation({
+    summary: 'Catch connection webhook responses',
+    description: 'Callback URL for connection'
+  })
+  @ApiResponse({ status: HttpStatus.CREATED, description: 'Created', type: ApiResponseDto })
+  async getConnectionWebhook(
+    @Body() connectionDto: ConnectionDto,
+    @Param('orgId') orgId: string,
+    @Res() res: Response
+  ): Promise<Response> {
+    connectionDto.type = 'Connection';
+    this.logger.debug(`connectionDto ::: ${JSON.stringify(connectionDto)} ${orgId}`);
+  
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove the commented code

Reply:
I have removed unused variables . Please review it